.travis.yml

```diff
@@ -12,13 +12,13 @@ before_script:
   - composer install --no-interaction --prefer-source --dev
   - ~/.nvm/nvm.sh install v0.6.14
   - ~/.nvm/nvm.sh run v0.6.14
+  - '[ "$TRAVIS_PHP_VERSION" != "7.0" ] || echo "xdebug.overload_var_dump = 1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini'
```

Comment:
I tried that in a previous PR: https://github.com/guzzle/guzzle/pull/1368
Build failed on all other versions because that line returns result code 1 in all other versions:
https://travis-ci.org/guzzle/guzzle/builds/103807788

|| ensures that 0 is returned in all versions.